data/core/units/monsters/Bear.cfg

```diff
@@ -16,7 +16,7 @@
     {AMLA_DEFAULT}
     cost=29
     usage=fighter
-    description= _ "Cave Bears are large, powerful beasts with sharp claws and large teeth. Adapted to roaming the dark caves, they have a keen sense of smell and hearing, but poor eyesight. When a rival predator, or some other threat, gets too close, a bear will reliably attack."
+    description= _ "Cave Bears are large, powerful beasts with sharp claws and large teeth. Adapted to roaming the dark caves, they have a keen sense of smell and hearing, but poor eyesight. When a rival predator, or some other threat, gets too close, a Bear will reliably attack."
```

Comment:
Acknowledging it's from the original text, but 'reliably' sounds odd to me. '...a Bear will _undoubtedly_ attack' sounds better to me.